src/bundle/Resources/public/js/scripts/helpers/tooltips.helper.js

```diff
@@ -67,7 +67,11 @@
             return;
         }
 
-        const tooltipNodes = baseElement.querySelectorAll(TOOLTIPS_SELECTOR);
+        const tooltipNodes = [...baseElement.querySelectorAll(TOOLTIPS_SELECTOR)];
+
+        if (!baseElement.isEqualNode(doc)) {
```

Comment:
I think that this check would be more precise:
```suggestion
        if (baseElement instanceof Element) {
```
ref.:
- types of nodes https://developer.mozilla.org/en-US/docs/Web/API/Node/nodeType
- node vs element (document vs element) https://stackoverflow.com/questions/9979172/difference-between-node-object-and-element-object

WDYT? Do you see any potential problems?